ci/jenkins/jobs/job-templates.yaml

```diff
@@ -163,3 +163,28 @@
         enabled: true
     triggers:
     - timed: '{timed}'
+
+- job-template:
+     name: '{name}-{test_name}-for-eks'
+     builders: '{builders}'
+     description: '{description}'
+     block-downstream: false
+     block-upstream: false
+     project-type: freestyle
+     properties:
+     - build-discarder:
+         artifact-days-to-keep: -1
+         artifact-num-to-keep: -1
+         days-to-keep: 7
+         num-to-keep: 30
+     - github:
+         url: https://github.com/vmware-tanzu/antrea/
+     scm:
+     - git:
+         branches:
+         - master
+         name: origin
+         refspec: +refs/heads/*:refs/remotes/origin/* +refs/pull/${ghprbPullId}/*:refs/remotes/origin/pr/${ghprbPullId}/*
```

Comment:
`${ghprbPullId}` is only available when the build is triggered from github pull request builder plugin. Maybe we can just delete the `refspec` line and let it use the default value.

Reply:
Done